### DISCUSSION TITLE
Web Server keeps consuming all memory?

### DISCUSSION THREAD

**Discussion by whillas-yabble** [dagster-io/dagster#23780] [Q&A]
The web server seems to progressively consume all memory and then need to be restarted?
![image](https://github.com/user-attachments/assets/2471d40e-bc0d-4065-b348-5aa276d79f54)
Are other ppl seeing this?

**Comment by garethbrickman** [comment]
Could be this known issue: https://github.com/dagster-io/dagster/issues/18997
